docs/docs/guides/build/assets/asset-selection-syntax/examples.md

```diff
@@ -0,0 +1,349 @@
+---
+title: 'Asset selection examples'
+sidebar_position: 100
+---
+
+This page contains common example asset selection queries and their implementation in Python, CLI, and the Dagster UI. For a full explanation of the filters, layers, operands, and functions that you can use to construct your own queries, see "[Asset selection syntax reference](reference)".
```

Comment:
This is the first page in the dropdown and immediately directs you to the second page, so maybe the reference should go first?

Reply:
Yeah, I can put the reference page first -- I put the examples page first in order to make the path examples as prominent as possible, but those should be easier to find now that they're at the top of the page, regardless of the page's position in the sidebar.

Reply:
Yeah I like this better!